### PR TITLE
Grant/cone get/drop command entitlement modification flag

### PR DESCRIPTION
**Pull request by GrantKop**
Implemented the --detailed flag that will display app information and entitlement information when using the get/drop commands

### CODE REVIEW

**Comment by GrantKop** [comment]
image
image

**Comment by GrantKop** [comment]
After last commit tables no longer show up

**Comment by GrantKop** [comment]
Updated output:
image

File: cmd/cone/app.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	return []string{
+		"App Name",
+		"Description",
+	}
+}
+
+func (r *App) Rows() [][]string {
+	return [][]string{{
+		client.StringFromPtr(r.app.DisplayName),
+
```
Comment (loganintech) [inline cmd/cone/app.go:29]: nit: space

File: cmd/cone/flags.go [hunk 2]
```diff
@@ -53,3 +54,7 @@ func addEmergencyAccessFlag(cmd *cobra.Command) {
 func addForceTaskCreateFlag(cmd *cobra.Command) {
 	cmd.Flags().Bool(forceFlag, false, "Force the creation of a task even if the user already has (or doesn't have) the entitlement.")
 }
+
+func addEntitlementDetailsFlag(cmd *cobra.Command) {
+	cmd.Flags().Bool(extraDetailsFlag, false, "Show extra details")
```
Comment (loganintech) [inline cmd/cone/flags.go:59]: "Show more details about the app and entitlement for this request"

**Review by loganintech — APPROVED** [review]

**Pull request merged**